src/devices/Board/PinUsage.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Board
+{
+    /// <summary>
+    /// Designated (or active) usage of a pin
+    /// </summary>
+    public enum PinUsage
```

Comment:
I'm confused. I though these will be covered by AlternatePinMode. Should those two abstractions be joined?